frontend-rsnt/src/components/DesktopOnlyCockBlock.tsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+type Props = {};
+
+export const DesktopOnlyCockBlock: React.FC<Props> = (props) => (
+  <div className="flex lg:hidden fixed z-50 bg-black w-full h-full flex-col items-center justify-center">
+    <div className="mb-40 text-30">(╯°□°)╯︵ ┻━┻</div>
+    <p className="text-center">
+      We are sorry, but Sonicboom currently supports only desktop devices <br />{' '}
+      Thank you for your understatement{' '}
```

Comment:
Delete this entire line. There is a grammatical mistake but also, it is redundant. 

`We are sorry, but Resonate currently only supports desktop devices`